aws/network/bastion/main.tf

```diff
@@ -22,13 +22,29 @@ variable "owner" {
   default = ""
 }
 
+variable "open_sesame" {
```

Comment:
First it was called "open_to_world" but then we thought it might sound horrifying.  But sure I can try to change it to a more descriptive name.